diff --git a/src/common/file_util.cpp b/src/common/file_util.cpp
index 46401bc19..218a0a519 100644
--- a/src/common/file_util.cpp
+++ b/src/common/file_util.cpp
@@ -12,7 +12,6 @@
 #include "common/common_paths.h"
 #include "common/file_util.h"
 #include "common/logging/log.h"
-#include "core/settings.h"
 
 #ifdef _WIN32
 #include <windows.h>
@@ -718,12 +717,8 @@ void SetUserPath(const std::string& path) {
 #endif
     }
 
-    g_paths.emplace(UserPath::SDMCDir, !Settings::values.sdmc_dir.empty()
-                                           ? Settings::values.sdmc_dir
-                                           : user_path + SDMC_DIR DIR_SEP);
-    g_paths.emplace(UserPath::NANDDir, !Settings::values.nand_dir.empty()
-                                           ? Settings::values.nand_dir
-                                           : user_path + NAND_DIR DIR_SEP);
+    g_paths.emplace(UserPath::SDMCDir, user_path + SDMC_DIR DIR_SEP);
+    g_paths.emplace(UserPath::NANDDir, user_path + NAND_DIR DIR_SEP);
     g_paths.emplace(UserPath::SysDataDir, user_path + SYSDATA_DIR DIR_SEP);
     // TODO: Put the logs in a better location for each OS
     g_paths.emplace(UserPath::LogDir, user_path + LOG_DIR DIR_SEP);
