From ad2fa69a8df1b1d20266a1712bb802235514e95e Mon Sep 17 00:00:00 2001
From: liushuyu <liushuyu011@gmail.com>
Date: Fri, 20 Aug 2021 04:46:35 -0600
Subject: [PATCH 3/4] Partially reverts PR 5759

---
 src/common/file_util.cpp | 9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

diff --git a/src/common/file_util.cpp b/src/common/file_util.cpp
index 7b360967b..0153c3914 100644
--- a/src/common/file_util.cpp
+++ b/src/common/file_util.cpp
@@ -12,7 +12,6 @@
 #include "common/common_paths.h"
 #include "common/file_util.h"
 #include "common/logging/log.h"
-#include "core/settings.h"
 
 #ifdef _WIN32
 #include <windows.h>
@@ -718,12 +717,8 @@ void SetUserPath(const std::string& path) {
 #endif
     }
 
-    g_paths.emplace(UserPath::SDMCDir, !Settings::values.sdmc_dir.empty()
-                                           ? Settings::values.sdmc_dir
-                                           : user_path + SDMC_DIR DIR_SEP);
-    g_paths.emplace(UserPath::NANDDir, !Settings::values.nand_dir.empty()
-                                           ? Settings::values.nand_dir
-                                           : user_path + NAND_DIR DIR_SEP);
+    g_paths.emplace(UserPath::SDMCDir, user_path + SDMC_DIR DIR_SEP);
+    g_paths.emplace(UserPath::NANDDir, user_path + NAND_DIR DIR_SEP);
     g_paths.emplace(UserPath::SysDataDir, user_path + SYSDATA_DIR DIR_SEP);
     // TODO: Put the logs in a better location for each OS
     g_paths.emplace(UserPath::LogDir, user_path + LOG_DIR DIR_SEP);
-- 
2.34.1

